appveyor.yml

```diff
@@ -25,15 +25,16 @@ environment:
       PYTHON_VERSION: "2.7.8"
       PYTHON_ARCH: "64"
 
-    - PYTHON: "C:\\Python34"
```

Comment:
I am afraid appveyor is too slow to support too many versions. We can still do a PR to trigger the build on a specific tag on more versions once we release the final version and upload the 3.4 packages manually to PyPI after the release.